datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -3186,6 +3196,22 @@ private void configureIngestionSourceResolvers(final RuntimeWiring.Builder build
                         })));
   }
 
+  private void configureExecutionRequestResolvers(final RuntimeWiring.Builder builder) {
+    builder
+        .type(
+            "ExecutionRequest",
+            typeWiring -> typeWiring.dataFetcher(
+                    "source", new LoadableTypeResolver<>(
```

Comment:
i'm wondering if it makes more sense to simply use the custom `GetIngestionSourceResolver` here instead of having `loadableTypes` have types that may not be entities. going forward we shouldn't make the same mistake as we did with ingestion sources, so this may be the only case we have to handle of an entity type not fitting into `loadableTypes`

although i am liking this flexibility of it being in `loadableTypes` as long as changing it so it doesn't have to be `EntityType` types doesn't seriously affect anything